### DISCUSSION TITLE
Incorrect response described in openapi schema and docs

### DISCUSSION THREAD

**Discussion by gimenete** [planetscale/discussion#414] [Bug Reports]
The open api spec says that `/organizations/{organization}/databases/{database}/deploy-requests` returns an object with a `data` attribute that's an array of strings. But it actually returns a lot more information.

The docs, probably generated from the openapi schema, also show that issue. Go to https://api-docs.planetscale.com/reference/list-deploy-requests and the example response looks like this:

<img width="478" alt="image" src="https://user-images.githubusercontent.com/50486/224577850-63ec5f0f-6bcf-4629-82ac-bd39b8d04ba9.png">

It could be that this other enpoint has the same issue https://api-docs.planetscale.com/reference/get-the-deploy-queue

**Comment by jgreet** [comment]
Thanks for the report, it looks a few other endpoints have the same spec/documentation issue. I've passed this on to the API team.
